.mailmap

```diff
@@ -223,3 +223,12 @@ Youssef Eweis <joey070@gmail.com> Youssef Eweis <Youssef15015@users.noreply.gith
 Rohith Varma Buddaraju <rohith.varma.buddaraju@gmail.com>
 Rohith Varma Buddaraju <rohith.varma.buddaraju@gmail.com> rohithvarma3000 <rohith.varma.buddaraju@gmail.com>
 Rohith Varma Buddaraju <rohith.varma.buddaraju@gmail.com> Rohith <rohith.varma.buddaraju@gmail.com>
+
+GARV PATIDAR <patidargarv04@gmail.com> 
+garvpatidar04 <patidargarv04@gmail.com>
+GARV PATIDAR <patidargarv04@gmail.com> garvpatidar04 <patidargarv04@gmail.com>
+garvpatidar04 <notgarvpatidar@gmail.com>
+GARV PATIDAR <notgarvpatidar@gmail.com> garvpatidar04 <notgarvpatidar@gmail.com>
```

Comment:
You could try`GARV PATIDAR <90318877+garvpatidar04@users.noreply.github.com>` to see if it works.

Reply:
Okay, I will try to do that.